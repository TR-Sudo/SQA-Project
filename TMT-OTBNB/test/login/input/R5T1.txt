Login
user026
user01
logout
quit