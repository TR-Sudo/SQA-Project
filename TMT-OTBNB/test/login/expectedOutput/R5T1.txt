Enter transaction: Enter username: User not found
Enter username: You have been logged in
Enter transaction: You have been logged out
Enter transaction: 